### PR TITLE
Week3 homework assignment

### PR DESCRIPTION
**Pull request by priyankapandey13**

### CODE REVIEW

File: week3/materials/homework-cron-lambda/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (paulinadubas) [inline week3/materials/homework-cron-lambda/.gitignore:1]: Good practice to add new line at the end of file :)

File: week3/materials/homework-cron-lambda/assignments.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
 #### Example asignment
```
Comment (paulinadubas) [inline week3/materials/homework-cron-lambda/assignments.md:1]: Answers look correct

**Review by paulinadubas — APPROVED** [review]
Looks correct

File: week3/materials/homework-cron-lambda/assignments.md (hunk 2 above)
Comment (priyankapandey13) [reply week3/materials/homework-cron-lambda/assignments.md:1]: Thanks 🙂

**Pull request merged**